Python/pywarpx/picmi.py

```diff
@@ -1065,6 +1066,9 @@ def initialize_inputs(self):
             pywarpx.warpx.gamma_boost = self.gamma_boost
             pywarpx.warpx.boost_direction = 'z'
 
+        if self.zmax_plasma_to_compute_max_step is not None:
```

Comment:
This check for None is not needed. Anything set to None is ignored when passing the parameters into WarpX.